Directory.Packages.props

```diff
@@ -11,5 +11,6 @@
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
     <PackageVersion Include="coverlet.collector" Version="3.2.0" />
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.2" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
```

Comment:
I guess **Microsoft.Extensions.DependencyInjection.Abstractions** would be sufficient, as only the interface `IServiceProvider` is required.

Reply:
hi @thoemmi 

I only use this package on the `UnitTests` project.

Reply:
Hi @thoemmi 

The `UnitTest` project uses the `Microsoft.Extensions.DependencyInjection` package.

We need to use the implementation of `IServiceProvider` in test methods.

Reply:
I get that the unit tests require the implementation of `IServiceProvider`. But MiniValidation itself only depends on `IServiceProvider`. I always try to keep package dependencies to a minimum. However, the PR is already approved, so my comment is dated :wink:

Reply:
For the record, adding the package to `Directory.Packages.props` doesn't make all projects in the solution depend on it. It just specifies the version for projects that **do** depend on it.